_posts/2018-06-27-inception-pipelines-pt4.markdown

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title:  "Seeds of Inception - Part 4"
+date:   2018-06-27
+categories: aws continuous deployment codepipeline codebuild inception pipeline
+author: Pete Yandell
+image: img/inception-pipelines/seed_germination.png
+---
+
+**Part 4 seeding a forest** - A theoretical example for DevOps on AWS
```

Comment:
Title case? Not sure if it matters on actual headings though (is everything capitalised?)